pom.xml

```diff
@@ -21,15 +21,15 @@
   <name>Event-B EMF Containment Root POM</name>
   <description>Root artifact for EMF Containment</description>
   <!-- REQUIRED: Version number (using semantic versioning) -->
-  <version>0.0.1.release</version>
+  <version>0.0.2.release</version>
   <packaging>pom</packaging>
   <parent>
     <!-- REQUIRED: Group ID is the feature ID -->
     <groupId>ac.soton.eventb.emf.containment</groupId>
     <!-- Name of the release engineering artifact -->
     <artifactId>releng</artifactId>
     <!-- REQUIRED: Version number of the release engineering artifact -->
-    <version>0.0.1.release</version>
+    <version>0.0.3.release</version>
```

Comment:
do we need to skip 0.0.2? (I don't think the version 0.0.2 was ever merged so is still part of this pull request?)

Reply:
I think we can go directly to 0.0.3 now. 0.0.2 actually is the same as 0.0.1. Due to an incorrect build of 0.0.1, I have to increase it to 0.0.2.

Reply:
Ok